### ISSUE TITLE
Feature request: enable multiple configs

### ISSUE THREAD

**Issue by Dachande663** [heldtogether/venice-php#1]
In the same way Monolog supports multiple drivers, it would be good if the system supported one or more configs e.g.

``` php
$factory = new Slab\Features\Factory;
$manager = new Slab\Features\Manager($factory);

$manager->addConfig(new Slab\Features\Configs\JSONFileConfig('my/global/features.json'));
$manager->addConfig(new Slab\Features\Configs\JSONFileConfig('my/global/branch-features.json'));
$manager->addConfig(new Slab\Features\Configs\DbConfig('features_table'));
```

Feel free to close as wontfix or v-whatever.

**Comment by heldtogether** [comment]
Good idea. 

It would be possible to write a class which conforms to `Slab\Features\Interfaces\ConfigInterface` which consolidates multiple config sources.

``` php
    class MultipleConfig implements Slab\Features\Interfaces\ConfigInterface {

        /**
         * Construct
         *
         * @param Slab\Features\Factory $factory
         * @param array $configs
         * @return void
         */
        public function __construct(Factory $factory, $configs) {

            $this->factory = $factory;
            $this->configs = $configs;

        }

        /**
         * Return an array of rules
         *
         * @return array
         */
        public function rules() {

            $rules = NULL;

            foreach ($this->configs as $config) {

                $rules = array_merge($rules, $config->rules());

            }

            return $rules;

        }

    }
```

However your interface is cleaner.

**Comment by heldtogether** [comment]
Implemented in de30824b22c15d2eac9b73bb4e0e3d0ebf10685c
